Hestia/Hestia/frontend/Local/UITableViewControllerLocalLogin.cs

```diff
@@ -0,0 +1,100 @@
+using Foundation;
+using System;
+using UIKit;
+using Hestia.DevicesScreen.resources;
+
+namespace Hestia
+{
+
+
```

Comment:
Whitelines

Reply:
Thanks for noticing, this is fixed in 6eaa73cb155275e6c6f2a40b802fe17bcf9bbf15